.github/workflows/workflow-lint.yml

```diff
@@ -0,0 +1,49 @@
+name: Workflow Lint
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - '*.yml'
+      - '.github/workflows/*.yml'
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '*.yml'
+      - '.github/workflows/*.yml'
+
+jobs:
+  main:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Install yamllint
+        run: |
+          pip install yamllint
+
+      - name: Run yamllint
+        run: |
+          yamllint -c .yamllint.yml -f github .github/workflows
+
+      - name: Run workflow lint script
+        run: |
+          node .github/scripts/lint-workflow.cjs
+        env:
+          GH_TOKEN: ${{ secrets.BOT_TOKEN }}
+
+      - name: Download actionlint
+        id: get_actionlint
+        run: |
+          bash <(curl \
+          https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+        shell: bash
```

Comment:
I'm a bit leery of a raw download and execute of a rolling file target like this. Can we target the bash script to a specific commit (preferably the commit corresponding to the tag for the release we're using) and target a specific version?

Reply:
> I'm a bit leery of a raw download and execute of a rolling file target like this. Can we target the bash script to a specific commit (preferably the commit corresponding to the tag for the release we're using) and target a specific version?

I think there's an even better option - it looks like the project has docker support, so we can run it out of docker without ever needing to grab the executable.